roles/confluent.kafka_connect/library/kafka_connectors.py

```diff
@@ -62,17 +68,76 @@ def get_current_connectors(connect_url):
             raise
         return []
 
+
 def remove_connector(connect_url, name):
     url = "{}/{}".format(connect_url, name)
     r = open_url(method='DELETE', url=url)
     return r.getcode() == 200
 
+
+# return value: success (bool), changed (bool), message (str)
 def create_new_connector(connect_url, name, config):
     data = json.dumps({'name': name, 'config': config})
     headers = {'Content-Type': 'application/json'}
     r = open_url(method='POST', url=connect_url, data=data, headers=headers)
```

Comment:
Not 100% sure, but all non-20x responses (e.g. 409) will be reported as exception, and will blow up. 

imho, all calls to `open_url` should be try-catched, as 

https://github.com/confluentinc/cp-ansible/blob/1409fd5e207c14c654aa05eab57748a9ee25b00d/roles/confluent.kafka_connect/library/kafka_connectors.py#L62-L69

fwiw, I've spotted this before and locally fixed some of these: https://github.com/confluentinc/cp-ansible/compare/6.0.0-post...jeqo:fix-401